hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/TestParquetToSparkSchemaUtils.java

```diff
@@ -18,8 +18,14 @@
 
 package org.apache.hudi.hive;
 
+import org.apache.hudi.avro.AvroSchemaUtils;
```

Comment:
Can you rename this file as you are including Avro in your testing.

Reply:
Arent you starting from Avro  in this testing ?

Reply:
The unit under test is the `ParquetToSparkSchemaUtils` method which takes in a `MessageType`. Not an Avro. The test uses Avro to generate the sample MessageType. This matches the runtime of the system which will request an avro schema, convert it to MessageType, and then call ParquetToSparkSchemaUtils.

The converter class (and then the test class to follow conventions) will be changed in https://github.com/apache/hudi/pull/14344